python/lsst/pipe/tasks/calibrateImage.py

```diff
@@ -880,45 +899,62 @@ def _fit_astrometry(self, exposure, stars):
         result = self.astrometry.run(stars, exposure)
         return result.matches, result.matchMeta
 
-    def _fit_photometry(self, exposure, stars, background):
+    def _fit_photometry(self, exposure, stars):
         """Fit a photometric model to the data and return the reference
         matches used in the fit, and the fitted PhotoCalib.
 
         Parameters
         ----------
         exposure : `lsst.afw.image.Exposure`
             Exposure that is being fit, to get PSF and other metadata from.
-            Modified to be in nanojanksy units, with an assigned photoCalib
-            identically 1.
+            Has the fit `lsst.afw.image.PhotoCalib` attached, with pixel values
+            unchanged.
         stars : `lsst.afw.table.SourceCatalog`
             Good stars selected for use in calibration.
+        background : `lsst.afw.math.BackgroundList`
+            Background model to convert to nanojansky units in place.
 
         Returns
         -------
         calibrated_stars : `lsst.afw.table.SourceCatalog`
             Star catalog with flux/magnitude columns computed from the fitted
-            photoCalib.
+            photoCalib (instFlux columns are retained as well).
         matches : `list` [`lsst.afw.table.ReferenceMatch`]
             Reference/stars matches used in the fit.
```

Comment:
It looks like we need to also add `matchMeta` to this output list, too. Sorry I'd neglected that.